sdk/storage/azure-storage-blob-nio/README.md

```diff
@@ -191,13 +191,6 @@ Write to a file. Only writing whole files is supported. Random IO is not support
 to guarantee that the data is available to be read.
 
 ```java readme-sample-writeToAFile
-try (OutputStream os = Files.newOutputStream(filePath)) {
-    os.write(0);
-}
-``` 
```

Comment:
nit: this looks like a bug in the codesnippet tooling where a trailing whitespace, possibly any character, in the closing code fence causes the code fence close to be ignored.